setup.py

```diff
@@ -174,154 +80,31 @@ def detect_encoding(readline):
             return "", []
     tokenize.detect_encoding = detect_encoding
 
-class InstallationError(Exception):
-    pass
-
-
-class CleanWrapper(clean):
-    """Wraps `python setup.py clean` to also cleans Gnulib installation"""
-    def run(self):
-        clean.run(self)
-        if include_gnulib:
-            print(">> Cleaning Gnulib build directory", file=sys.stderr)
-            try:
-                check_call(["make", "clean"], cwd=GNULIB_BUILD_DIR)
-            except CalledProcessError:
-                print(">> WARNING: Failed to clean Gnulib build!", file=sys.stderr)
-
-
-class BuildScriptWrapper(build_scripts):
-    """Override the script-building machinery of distutils.
+source_files = ["src/genomedata_load_data.c"]
```

Comment:
If we can rename `_load_data.py` to `load_data.py` then please rename this to `_load_data.c`

Reply:
Can you rename it to `_c_load_data.c`?